students/Matt_Hudgins/Lesson08/circle.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+# chmod +x circle.py needs to be performed before executable
+
+'''
+    File Name: test_html_render.py
+    Author: Matt Hudgins
+    Date created: 6/3/18
+    Date last modified: 6/3/18
+    Python Version 3.6.4
+'''
+
+from math import pi
+
+class Circle():
+    """This function creates a class for circles"""
+    def __init__(self, radius):
+        self.radius = radius
+
+    @property
+    def radius(self):
+        return self.radius
+
+    @radius.setter
+    def raduius(self, value):
+        if value < 0:
+            raise ValueError('Must have a value greater than 0')
+        self.radius = value
```

Comment:
In this case, I think it'd be simpler to have `radius` be an instance attribute and handle the validation in the `__init__` method